src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,7 @@
+* {
+  margin: 0;
+}
```

Comment:
You already do general styling in another file, so it's worth keeping it in one place and not splitting the use of the same selectors in different files.